curvefs/src/client/inode_wrapper.cpp

```diff
@@ -350,5 +396,34 @@ InodeWrapper::UpdateInodeStatus(InodeOpenStatusChange statusChange) {
     return CURVEFS_ERROR::OK;
 }
 
+// TODO(wuhanqing): update extents incrementally
+CURVEFS_ERROR InodeWrapper::SyncVolumeExtents() {
```

Comment:
We have an interface for SyncVolumeExtents, why SyncFullInode doesn't just update other parts？

Reply:
SyncVolumeExtents is unused and is been deleted.

But, I think this class is going to be unmaintainable, it has different responsibilities, and somewhere use this interface and somewhere else, but may do the same thing. And, it has to know the data backend.